flag-app/flag-app-project/src/components/BorderCountry.tsx

```diff
@@ -0,0 +1,28 @@
+import { useDarkMode } from "../hooks";
+import { Country } from "../interfaces";
+
+interface Props {
+  country: Country;
+}
+
+export const BorderCountry = ({ country }: Props) => {
+  const { mode } = useDarkMode();
+  const { borders } = country;
+
+  if (!borders) return <p>Country has no borders</p>;
+
+  return (
+    <>
+      {borders.map((border) => (
+        <div
+          className={`border shadow-md px-6 ${
+            mode === "dark" ? "shadow-white" : ""
```

Comment:
And here - even though it's not so long, it's good to (at least) establish a variable for something like a "base class name"